server/lib/dev_tool/templates/fake_hydra/o_auth2/choose-user.html.heex

```diff
@@ -0,0 +1,15 @@
+<% base_url = DevTool.FakeHydra.Router.Helpers.o_auth2_path(@conn, :user) %>
```

Comment:
You should rename the folder "o_auth2" to "oauth2" to keep the same naming across the project.